mat64/list_test.go

```diff
@@ -1104,9 +1120,9 @@ func testTwoInput(t *testing.T,
 					// Ensure that b is the correct transpose type if applicable.
 					// The receiver is always a concrete type so use it.
 					bSame := receiver
-					u, ok = bSame.(Untransposer)
+					u, ok = b.(Untransposer)
```

Comment:
bSame is set to the receiver, but the code for aSame has made receiver in its base form. bSame is never an untransposer, since receiver isn't. We do care if b is a transposer so we re-transpose it correctly.